python/cuspatial/cuspatial/_lib/interpolate.pxd

```diff
@@ -0,0 +1,24 @@
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+# distutils: include_dirs = cuspatial/bindings/
+
+from cudf._libxx.table cimport *
+
+cdef extern from "cubic_spline.hpp" namespace "cuspatial" nogil:
+    cdef unique_ptr[table] cpp_cubicspline_coefficients "cuspatial::cubicspline_coefficients" (  # NOQA: E501
```

Comment:
People have been using `\` to break the line rather than `NOQA` but no strong opinions from me